### ISSUE TITLE
"failed to spawn" if perl not installed

### ISSUE THREAD

**Issue by Tintinabula** [h2o/h2o#288]
The base install of FreeBSD does not include perl. If perl is not installed and 
H2O is started a generic, "failed to spawn" error is shown, but the cause of 
the error is not clear.

```
[root@FreeBSD10 ~]# uname -a
FreeBSD FreeBSDzfs 10.1-RELEASE-p9 FreeBSD 10.1-RELEASE-p9 #0: Tue Apr  7 01:09:46 UTC 2015     root@amd64-builder.daemonology.net:/usr/obj/usr/src/sys/GENERIC  amd64

[root@FreeBSD10 ~]# which perl

[root@FreeBSD10 ~]# /usr/local/bin/h2o --mode=daemon -c /usr/local/etc/h2o.conf
failed to spawn /usr/local/share/h2o/start_server:No such file or directory
```

Once perl is installed the server starts without error.

```
[root@FreeBSD10 ~]# pkg install perl5

[root@FreeBSD10 ~]# which perl
/usr/bin/perl

[root@FreeBSD10 ~]# /usr/local/bin/h2o --mode=daemon -c /usr/local/etc/h2o.conf
start_server (pid:29222) starting now...
```

**Comment by kazuho** [comment]
Sorry the issue has been closed unexpectedly; it has been fixed in the upstream but the version bundled with H2O has not yet been updated.

**Comment by kazuho** [comment]
merged to master in #289.
